### PR TITLE
Don't trigger codecov reports

### LINKED ISSUE #919: Consider ditching coverage
The codecov.io comments aren't currently being useful: they tend to report coverage diffs in unrelated files. It may make sense to ditch the coverage reports.

See for example #917, which reports coverage changes in unrelated files.

Core Python is currently having a similar discussion. See https://bugs.python.org/issue39704

### PR DESCRIPTION
**Pull request by mdickinson**
This PR removes the codecov triggers from the Travis CI configuration.

Fixes #919

### CODE REVIEW

**Comment by kitchoi** [comment]
Thanks. There is one from `appveyor.yml` too.

**Comment by mdickinson** [comment]
Thanks, fixed. I wonder whether the dual reports could be part of what's confusing codecov.io. Someday we should try to re-implement and make this useful. But not today.

File: appveyor.yml [hunk 1]
```diff
@@ -20,11 +20,8 @@ install:
   - install-edm-windows.cmd
   - edm install -y wheel click coverage
   - edm run -- python etstool.py install --runtime=%runtime%
+
```
Comment (mdickinson) [inline appveyor.yml:23]: Drive-by spacing consistency nitpick

**Comment by mdickinson** [comment]
AAARGH!

**Comment by kitchoi** [comment]
Hmmm, that's puzzling.

**Comment by kitchoi** [comment]
Maybe because master still has the codecov setup, this needs to be merged first for this to take effect?

**Comment by kitchoi** [comment]
(it was a baseless wild guess)

**Comment by mdickinson** [comment]
No idea. Let's merge and see what happens.
